adcircpy/server/_driver_file.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, driver):
         self._driver = driver
 
     def write(self, path, overwrite=False):
-        with open(pathlib.Path(path), 'w') as f:
+        with open(pathlib.Path(path), 'w', newline='\n') as f:
```

Comment:
Why do we need this? (just curious)

Reply:
when writing the job scripts from my windows machine it output windows CRLF (yuck)

Reply:
Does CRLF works with ADCIRC? If yes, then this is not necessary. If you are annoyed by this, you should probably switch to a POSIX environment.

Reply:
I got errors on TACC when running from a configs with CRLF, I had to run `dos2unix` to get it to work.

Reply:
Makes sense, because it is a bash file... This should not be a problem for fort.* files.